packages/toggle/src/component.tsx

```diff
@@ -38,18 +43,44 @@ export function Toggle(props: ToggleProps) {
 
   const helpId = props.helpText ? `${id}__hint` : undefined;
   const isInvalid = props.invalid;
-  const isRadioGroup = props.type === 'radio' || props.type === 'radio-button';
-
+  const isRadio = props.type === 'radio';
+  const isCheckbox = props.type === 'checkbox';
+  const isRadioButton = props.type === 'radio-button';
+  const isRadioGroup = isRadio || isRadioButton;
   const isControlled =
-    props.selected !== undefined || props.checked !== undefined;
+    props.selected !== undefined || props.checked !== undefined;  
+
+  const labelClasses = classNames({
+    [ccToggle.label]: !isRadioButton,
+    [ccToggle.focusable]: !isRadioButton,
+    [ccToggle.noContent]: !props.indeterminate,
+    [ccToggle.indeterminate]: props.indeterminate,
+    [`${ccToggle.radio} ${ccToggle.labelRadioColors}`]: isRadio,
+    [ccToggle.radioInvalid]: isRadio && isInvalid,
+    [`${ccToggle.checkbox} ${ccToggle.labelCheckboxColors}  ${ccToggle.checkboxChecked}`]: isCheckbox,
+    [ccToggle.icon]: isCheckbox && !props.indeterminate,
+    [ccToggle.checkboxInvalid]: isCheckbox && isInvalid,
+    [ccToggle.scLabel]: isRadioButton,
+    [ccToggle.scLabelJustified]: props.equalWidth,
+    [ccToggle.scLabelSmall]: props.small,
+  });
+  const inputClasses = classNames({
+    [ccToggle.input]: true,
+  });
+
+  const wrapperClasses = classNames(props.className, {
+    [ccToggle.wrapper]: true,
+    [`${ccToggle.segmentControl} ${ccToggle.focusableWithin}`]: isRadioButton,
+    [ccToggle.scJustified]: props.equalWidth,
```

Comment:
I don't think this will work because it sets flex on a container that already has inline-flex (`ccToggle.wrapper`). I think these two classes might need to be applied interchangeably.

Reply:
You are correct, this is needed to be set on a `group` and that(`group` that is) is added in `Item`. So this is the part that I'll fix for the radio buttons when this is merged. 
This part of the toggle is not yet finished 👇 But will be in the next PR
![image](https://user-images.githubusercontent.com/37986637/236753556-a32ccee5-c0bf-4982-875b-fa5b74c730fb.png)